gobblin-service/src/main/java/org/apache/gobblin/service/monitoring/DagActionStoreChangeMonitor.java

```diff
@@ -94,6 +103,9 @@ public DagActionStoreChangeMonitor(String topic, Config config, DagActionStore d
     this.flowCatalog = flowCatalog;
     this.orchestrator = orchestrator;
     this.isMultiActiveSchedulerEnabled = isMultiActiveSchedulerEnabled;
+    // instantiating using default ctor; subsequent PR will handle instantiating with multi-args ctor
+    this.dagTaskStream = new DagTaskStream();
+    this.isRefactoredDagManagerEnabled = ConfigUtils.getBoolean(config, ServiceConfigKeys.GOBBLIN_SERVICE_DAG_MANAGER_ENABLED_KEY, false);
```

Comment:
I'd rather you rename this variable as `isMultiLeaderDagManagerEnabled` otherwise the question becomes why didn't we just modify the base classes.